### DISCUSSION TITLE
Compatibilty Matrix

### DISCUSSION THREAD

**Discussion by Wildshire** [kubernetes-sigs/scheduler-plugins#791] [Q&A]
Hello

I have just installed k8s with v1.30.3 and I was just wondering when the plugins are going to be compatible with this k8s version. Or if perhaps someone has tried to install the plugins with latest version and was able to do it.

[This](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master?tab=readme-ov-file#compatibility-matrix) is what I checked.

Thanks in advance

**Comment by Huang-Wei** [comment]
v1.30 will be released in ~3 months. You can use v1.29 as long as your usecase don't use the scheduling-related APIs **_firstly_** introduced in v1.30.
